### ISSUE TITLE
Is it possible to get all event index in event align

### ISSUE THREAD

**Issue by manburst** [jts/nanopolish#959]
Hello sir 
I have some questions about nanopolish event align
1. Can we get all event_index in output? because we get only the event index that is present in seqeunce. If it is possible what is the option to get all of them?
2. How do we check each event index to the signal. To clarify, I want to check my sequence in fasta file by comparing each 5-mer to each event index but I don't get where my first or last event index starts in the plot of current data. I have read your blog post, each event index is current in range of 0.5 second right? But in the plot, the information we get on X-axis is current sample index.

Thanks in advance.
I have to apologize if my question is not clear. Inform me, I will clarify more.
Thanks again.

**Comment by jts** [comment]
Hi,

1. Unfortunately this is not possible. `nanopolish eventalign` is based on signal-to-basecall and basecall-to-reference alignments. Hence if some bases of the read are not aligned to the reference they will be skipped.
2. You can try the `--signal-index` option which will print out the index of the raw sample for each aligned event.

**Comment by manburst** [comment]
Thanks for the quick reply!
Now I have re-analysis the read again and have found that event align might align the sequence to signal too early?
Here is the plot of my signal that indicated event align start location
![image](https://user-images.githubusercontent.com/68594551/142562719-90e61a83-965a-4752-95d7-df2508126666.png)
You will notice the long homopolymer which is our experiment that add them at 3' end before ligate to sequencing adaptor.
By the way, as I mentioned that event_align tries to align the signal upstream of added 3' homopolymer. 
Is this normal or something I have to adjust to option? Or am I misunderstanding something?
Here is the table of event_align at 3' end of the read.
![image](https://user-images.githubusercontent.com/68594551/142563003-1c41f17f-ee44-4cad-8865-3e9d1d53ac07.png)
Thanks again

**Comment by jts** [comment]
Based on the k-mer size I guess this is RNA? If so, the signal analysis coming out of nanopolish can be a little complicated as the sequencing is in the 3' -> 5' direction but nanopolish flips the events to be 5' -> 3' direction. I note that the sample indices (`start_idx`, `end_idx`) are decreasing, so perhaps the aligned signal goes from the highlighted region towards the lowest coordinate, rather than across the long homopolymer region? Would this make more sense.

**Comment by manburst** [comment]
Yes, this is RNA data and I already noticed that nanopolish filp from 3' -> 5' to 5-> 3' but I still wonder why start_index is started before 3 ' end homopolymer. It should be started after that homopolymer.
Is it because the highlighted region has a current level similar to reference K-mer?
